### DISCUSSION TITLE
Download previous versions

### DISCUSSION THREAD

**Discussion by stdedos** [winsiderss/systeminformer#1580] [Questions / Answers] [answered]
Hello there,

I have recently updated, and I am facing the following "undesired situations":

1. BSOD [minidump.zip](https://github.com/winsiderss/systeminformer/files/10521684/minidump.zip) (yes, ik, my problem - shouldn't install an anti-virus)
2. Status bar is missing
3. Window graphs are missing

and that was all good and fine - I'll just downgrade.

... but

4. I don't have the previous versions of the software, neither can I find any of them in https://systeminformer.sourceforge.io/nightly.php / https://ci.appveyor.com/project/winsiderss/systeminformer/history

- Can I "somehow find out" which version I had installed?
- Are the aforementioned issues my problem?
- Does anyone have older versions available they could share? I have the last PH; but I was hoping not to downgrade "that far"

**Comment by dmex** [comment, marked as answer]
> BSOD minidump.zip

Dump files should be uploaded to #1523 

> yes, ik, my problem - shouldn't install an anti-virus

The crash only affects people running old versions of Windows 10 from more than 5 years ago (1809 and older).

> Are the aforementioned issues my problem?

We're waiting for Microsoft.... Until they fix the issue your choices are either disable the kernel driver or update Windows 10. 

> Can I "somehow find out" which version I had installed?

You could try checking the file versions?

> don't have the previous versions of the software, neither can I find any of them

Nightly builds are uploaded here: 
https://github.com/winsiderss/si-builds/releases

**Reply by stdedos** [reply]
So, downgrading to Version: 3.0.5568 (88476fc):
1. Also caused a BSOD (while "unloading the https://github.com/winsiderss/systeminformer/issues/1523 driver"?)
2. Returned the "Status bar", the "File Menu bar", and the small "Graphs Bar"

Also btw, I am on Windows 1909! (also mentioned that on the ticket)

Are the `"Status bar", the "File Menu bar", and the small "Graphs Bar"` part of the https://github.com/winsiderss/systeminformer/issues/1523?
Or are they some feature depreciations I've missed?

**Reply by dmex** [reply]
> Or are they some feature depreciations I've missed?

All these features are still there and haven't changed. For whatever reason your installation is missing one of the plugins (likely getting deleted by your Avast AV or something).

>  I am on Windows 1909! (also mentioned that on the ticket)

The next update blocks the driver for anything below 20H1. You won't be able to enable the driver with 1909 until Microsoft fix the issue.
